context-free.md

```diff
@@ -2,21 +2,55 @@
 
 ##  Who is this programming language for?
 
+Context Free is useful for people who are interested in creating
+computer-generated art.  This could have uses for graphical designs or just
+expressing creativity.  Other features in Context Free could interest animators
+as well.  
 
 ## What is easy to do in this language? Why is it easy?
 
+In Context Free, it is very easy to draw art recursively.  This allows the
+creator to make fractals and other patterns with relatively little work.  This
+is made possible since shapes can include themselves in their own description, 
+so Context Free can make complicated images that would be very difficult to
+draw.  In addition, Context Free is good at incorporating randomness in images.
```

Comment:
This is something I wish I'd realized for my own Context Free program. Could you elaborate or give examples on where randomness would come up in a Context Free artwork? I would like to know if it's randomness due to errors or randomness that we could deliberately introduce, kind of like what Ellsworth Kelly did with his _Spectrum of Colors Ordered by Chance_ series.